unlock-app/src/components/interface/checkout/main/Guild.tsx

```diff
@@ -48,25 +48,30 @@ export function Guild({ injectedProvider, checkoutService }: Props) {
 
   const disabled = data && data.filter((d) => !d).length > 0
 
+  const guildUrl =
+    guild && guild.urlName ? `https://guild.xyz/${guild?.urlName}` : '#'
+
+  const guildName = guild && guild?.name ? guild.name : undefined
```

Comment:
there is no need to do this right. guild?.name itself is enough. It will be undefined if name property or guild does not exist.